libraries/AP_HAL_ChibiOS/hwdef/AeroFox-Airspeed/hwdef.dat

```diff
@@ -0,0 +1,24 @@
+include ../f103-periph/hwdef.inc
+
+define CAN_APP_NODE_NAME "org.ardupilot.AeroFox_Airspeed"
+
```

Comment:
Should probably a board ID of your own; you can do that in `Tools/AP_Bootloader/boards_types.txt`.  Somewhere above 1076 just add a line like the others in there and that number in here.

Reply:
Sorry, by the way our airspeed periph is not flight control. Do we need to add a board ID?

Reply:
if you already have customers using this airspeed sensor with the current board ID then it may be better to leave it on that ID so they can update to the new firmware easily. Otherwise they won't be able to update using CAN GUI tool